### PR TITLE
pages.pt_PT: update outdated pages

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_PT/common/tmux.md [hunk 1]
```diff
@@ -9,19 +9,15 @@
 
 - Inicia uma sessão com nome:
 
-`tmux new-session -s {{nome}}`
+`tmux new- -s {{nome}}`
```
Comment (waldyrious) [inline pages.pt_PT/common/tmux.md:12]:
```suggestion
`tmux new -s {{nome}}`
```

**Comment by waldyrious** [comment]
Thanks for doing this update, @sebastiaanspeck! I found only a small typo in the diff, as I pointed out in an inline comment. Otherwise these changes look good, although some of the removed examples (specifically, those from `touch`) were actually adapted rather than removed in the original. That said, I don't have the time to review and add all the missing content, so I'll leave that as low-hanging fruit for the next Portuguese-speaking contributor :)

**Review by kbdharun — APPROVED** [review]

File: pages.pt_PT/linux/gedit.md [hunk 2]
```diff
@@ -5,16 +5,16 @@
...
 - Abre múltiplos ficheiros de texto:
 
-`gedit {{ficheiro1 ficheiro2}}`
+`gedit {{caminho/para/ficheiro1 caminho/para/ficheiro2 ...}}`
 
-- Mostra a lista de formatações de texto disponíveis:
+- Abre um ficheiro de texto com uma formatação específica:
 
-`gedit --list-encodings`
+`gedit --encoding={{formatação}} {{caminho/para/ficheiro}}`
```
Comment (isaacvicente) [inline pages.pt_PT/linux/gedit.md:16]:
```suggestion
`gedit --encoding={{UTF-8}} {{caminho/para/ficheiro}}`
```

File: pages.pt_PT/osx/systemsetup.md [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 - Ativa o serviço de hora de rede com um fuso horário e servidor específico:
 
-`systemsetup -settimezone {{Europe/Lisbon}} -setnetworktimeserver {{2.pt.pool.ntp.org}} -setusingnetworktime on`
+`systemsetup -settimezone "{{Europe/Lisbon}}" -setnetworktimeserver {{2.pt.pool.ntp.org}} -setusingnetworktime on`
 
 - Colaca a máquina sem dormir, reiniciando automaticamente em falta de energia ou pânico do núcleo do sistema:
```
Comment (isaacvicente) [inline pages.pt_PT/osx/systemsetup.md:14]:
```suggestion
- Coloca a máquina sem dormir, reiniciando automaticamente em falta de energia ou pânico do núcleo do sistema:
```

**Review by isaacvicente — COMMENTED** [review]
Hi! There was just a typo and an argument missing, as compared with the English page.

Everything else seems fine to me.

**Comment by isaacvicente** [comment]
> > Hi! There was just a typo and an argument missing, as compared with the English page.
> > Everything else seems fine to me.
> 
> Thank you for spotting those issues. I have fixed them. Could you please review and approve it if it looks good to you.

Sure :)

**Review by isaacvicente — APPROVED** [review]

**Pull request merged**